enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -63,11 +84,12 @@
     (let [extraction (atom nil)
           entities   (atom nil)]
       (ts/with-source-and-dest-dbs
-        ;; TODO Generating some nested collections would make these tests more robust.
+        ;; TODO Generating some nested collections would make these tests more robust, but that's difficult.
```

Comment:
Dumb question but is it really that difficult to generate nested Collections?

Reply:
I think there would be a way to do a "prefixed" style like how tables 10-19 belong to database 1, etc.
However I've tried twice now and then given up. It's just not a relationship specmonstah's generators are designed to capture.